README.md

```diff
@@ -37,7 +37,25 @@ Enable the API call on deletion:
 TechDivision:
   Jobs:
     GoogleApi:
-      enableApiCallOnJobDeletion: true
+      options:
+        enableApiCallOnJobDeletion: false
+        ...
+```
+
+After configuration there are a few things you need to do, if you haven't done them already:
+[Prerequisites for the Indexing API](https://developers.google.com/search/apis/indexing-api/v3/prereqs#verify-site) 
+
```

Comment:
We should add some documentation here that the verification must set the service account as owner for the property